common/src/main/java/com/wynntils/features/ui/CustomTradeMarketResultScreenFeature.java

```diff
@@ -23,6 +25,7 @@
 import net.minecraft.world.item.ItemStack;
 import net.minecraftforge.eventbus.api.SubscribeEvent;
 
+@ConfigCategory(Category.UI)
```

Comment:
Not trade market?

Reply:
Well all custom screens are here, as well as the feature itself is in this package

Reply:
True, just most people I've seen searching to disable it have expected it to be in trade market category

Reply:
Kind of have to rework categories either way..